### PR TITLE
*dabs on stocks*

### PR DESCRIPTION
**Pull request by Amelia0010**
### Intent of your Pull Request

Includes the hardcap we put on stock value on prebase yogs, also includes the fix for the spiking we had. This is the stockcode directly before the removal, so it doesn#t include the two PRs that were opened after the removal PR but merged before, which is a good thing since one is a revert of the other.

#### Changelog

:cl:  
rscadd: Stocks are back!
/:cl:

### CODE REVIEW

File: _maps/map_files/YogStation/YogStation.dmm [hunk 1]
```diff
@@ -28859,6 +28859,10 @@
 	pixel_x = -24
 	},
 /obj/item/folder/yellow,
+/obj/item/storage/firstaid/regular{
```
Comment (Moja10) [inline _maps/map_files/YogStation/YogStation.dmm:28862]: This is the real important addition of this PR. I missed you so much cargo medkit

Comment (Amelia0010) [reply _maps/map_files/YogStation/YogStation.dmm:28862]: it's already there, I just moved it.

File: _maps/map_files/YogStation/YogStation.dmm [hunk 2]
```diff
@@ -29885,10 +29889,7 @@
 /area/quartermaster/storage)
 "bvZ" = (
 /obj/structure/table,
-/obj/item/storage/firstaid/regular{
```
Comment (Amelia0010) [inline _maps/map_files/YogStation/YogStation.dmm:29888]: see:

**Comment by Nickvr628** [comment]
Hurray, I can’t wait to get 500,000 credits as cargo again!

**Comment by Amelia0010** [comment]
image
😎

**Pull request merged**